src/main/java/uk/gov/hmcts/darts/audio/controller/AudioController.java

```diff
@@ -77,12 +80,62 @@ public ResponseEntity<Resource> preview(Integer mediaId) {
         return new ResponseEntity<>(new InputStreamResource(audioMediaFile), HttpStatus.OK);
     }
 
+    @SneakyThrows
+    @Override
+    @SecurityRequirement(name = SECURITY_SCHEMES_BEARER_AUTH)
+    @Authorisation(contextId = MEDIA_ID,
+        securityRoles = {JUDGE, REQUESTER, APPROVER, TRANSCRIBER, LANGUAGE_SHOP_USER, RCJ_APPEALS})
+    public ResponseEntity<byte[]> preview2(Integer mediaId, @RequestHeader(value = "Range", required = false) String httpRangeList) {
```

Comment:
Should this be `range` to match the audio.yaml `in: header`?

Reply:
I'm not sure it makes a difference

Reply:
Sorry, think what you have here is correct - it was the yaml that needed to be the same case, then no need to override the generated param stuff. That's all I was meaning.